### PR TITLE
[Downloader] Add support for git repo modules

### PR DESCRIPTION
**Pull request by Aioxas**
Referencing #595

### CODE REVIEW

**Comment by Jan200101** [comment]
Seems okay I would suggest checking if it starts with git+ and doesnt just contain it.

File: cogs/downloader.py [hunk 1]
```diff
@@ -508,6 +508,8 @@ def get_dir_name(self, url):
         return git_name[:-4]
 
     def is_lib_installed(self, name):
+        if "git+" in name:
```
Comment (Jan200101) [inline cogs/downloader.py]: Would suggest changing it to
`if name.startswith("git+"):`

But otherwise its fine

**Review by Jan200101 — APPROVED** [review]

**Comment by Jan200101** [comment]
Finding the name over the git link may be a issue as some modules have other names in their github links then the one that need to be imported

**Comment by Aioxas** [comment]
Can you give me an example of one of those.

**Comment by Jan200101** [comment]
[Python Valve](https://github.com/Holiverh/python-valve) its github link links to python-valve but its imported using `import valve`

**Comment by Aioxas** [comment]
Wait a second, installing using git+ link is one thing. It's up to you to have the correct name of the package you are importing. So there's no need to change anything.

**Comment by Jan200101** [comment]
Im gonna give it a try and test if it works. Could be that a brain fart

**Comment by Jan200101** [comment]
yep brain fart.
It works

**Review by Jan200101 — APPROVED** [review]

**Comment by tekulvw** [comment]
I'm gonna nix this because there's no dire need to implement it in V2 and it's pointless in Downloader V3

**Pull request closed without merging**